Twig/Extension/DeprecatedTextExtension.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Twig\Extension;
+
+/**
+ * NEXT_MAJOR : remove this class and the twig/extensions dependency.
```

Comment:
source? I think we already discussed this somewhere, but I'm not sure what the outcome was…